wsd/ClientSession.hpp

```diff
@@ -376,6 +376,7 @@ class ClientSession final : public Session
     int _tileHeightPixel;
     int _tileWidthTwips;
     int _tileHeightTwips;
+    double _dpiScale;
```

Comment:
This _dpiScale member isn't used (and same for local var dpiScaleStr in ClientSession.cpp). I imagine we don't actually need those if we are just passing through the message from CanvasTileLayer.js?